dataset.py

```diff
@@ -160,9 +160,8 @@ def pull_item(self, index):
         img, boxes, labels = self.transform(
             img, self.phase, anno_list[:, :4], anno_list[:, 4])
 
-        # change channel color order from BGR to RGB
-        # and convert order from (h, w, c) to (c, h, w)
-        img = torch.from_numpy(img[:, :, (2, 1, 0)]).permute(2, 0, 1)
+        # convert order from (h, w, c) to (c, h, w)
+        img = torch.from_numpy(img).permute(2, 0, 1)
```

Comment:
Do you mean that all incoming image format is RGB?

Reply:
sorry for lacking explanation, [here](https://github.com/abeja-inc/platform-template-object-detection/pull/12/files#diff-b24696c07a0a52292a89d29937161fcfR126) the image is converted to RGB format.

Reply:
LGTM